content/ripplerest_api.md

```diff
@@ -1530,13 +1224,20 @@ Optionally, you can also include the following query parameters:
 
 If the length of the `payments` array is equal to `results_per_page`, then there may be more results. To get them, increment the `page` query paramter and run the request again.
 
-*Note:* It is not more efficient to specify more filter values, because Ripple-REST has to retrieve the full list of payments from the `rippled` before it can filter them.
```

Comment:
It wasn't removed. It was just moved. https://github.com/ripple/ripple-dev-portal/pull/113/files#diff-f7143e9cdcfd43266bc9e20b8dec1c36R1235